services/webnn/webnn_context_provider_impl.cc

```diff
@@ -246,17 +248,27 @@ void WebNNContextProviderImpl::CreateWebNNContext(
       return;
     }
 
-    scoped_refptr<ort::AllocatorOrt> allocator_ort =
-        ort::AllocatorOrt::GetInstance();
-    if (!allocator_ort) {
+    // `OrtEnv` is reference counted. The first `CreateEnv()` will create the
+    // `OrtEnv` instance. The following invocations return the reference of the
+    // same instance.  It is released upon the last reference is removed via
```

Comment:
> It is released upon the last reference is removed via `ReleaseEnv()`

Do you mean each `ReleaseEnv` call will reference -1, when reference == 0, the `env` is released?

Reply:
Right, see https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/core/session/ort_env.cc#L95